assets/tests/components/mapPage.test.tsx

```diff
@@ -826,14 +832,10 @@ describe("<MapPage />", () => {
 
           const selectedVehicle = shuttleFactory.build()
 
-          const shape = shapeFactory.build({
-            stops: stopFactory.buildList(8),
-          })
-
           setHtmlWidthHeightForLeafletMap()
           mockUseVehiclesForRouteMap({})
           mockUseVehicleForId([selectedVehicle])
-          mockUseTripShape({ [selectedVehicle.tripId!]: shape })
+          mockUserRoutePatternsByIdForVehicles([]) // no route pattern for shuttle vehicle
```

Comment:
note: btw, I think I wrote this, and if I remember correctly, I was providing a real route pattern for this test so that if the map rendered the route pattern for some reason, the assertion would fail[^1] if the map invalidated this. My understanding of the map component before was that if there's no shape to render, even if the map tried to render a shape, we wouldn't know or be able to fail the test.


[^1]: this assertion
```
expect(
    container.querySelector(".m-vehicle-map__route-shape")
).not.toBeInTheDocument()
```